L1Trigger/VertexFinder/src/VertexFinder.cc

```diff
@@ -1242,26 +1250,45 @@ namespace l1tVertexFinder {
     // 1 top PV from BRS Chosen:
     // vertices_.emplace_back(vertices.at(pv->second));
     // 1 top PV from Chris' network:
-    vertices_.emplace_back(vertices.at(nnChosenPV->first));
+    vertices_.emplace_back(vertices.at(nnChosenPV->first).z0());
```

Comment:
I had been putting into vertices_ an object of type RecoVertex, but now you are bring to put a float in?